intellij-plugin-verifier/verifier-core/src/main/java/com/jetbrains/pluginverifier/verifiers/method/KotlinMethods.kt

```diff
@@ -137,11 +155,64 @@ object KotlinMethods {
     val isAParent = method.containingClassFile.interfaces.any {
       it == actualKotlinOwner
     }
-    if (!isAParent) {
+    @Suppress("RedundantIf") // for reading clarity
+    if (!isAParent && !isImplementingAParentInterface) {
       return false
     }
 
     // The method is a kotlin default method
     return true
   }
-}
\ No newline at end of file
+
+  /**
+   * Check when an interface extends another the Idea interface that has private types.
```

Comment:
Unfortunately I can't parse this sentence

Reply:
Sorry I'm not sure how to phrase that correctly. Would that be easier to read?

> Checks whether this method is forwarding to the `DefaultImpls` of the parent interface.

Reply:
Done